### DISCUSSION TITLE
Azure SQL Managed Instance Maintenance schedule

### DISCUSSION THREAD

**Discussion by badkernel** [olahallengren/sql-server-maintenance-solution#730] [Questions & Answers]
Hello all

Did anyone used Ola's solutions with azure automation (runbooks) and if so do you have examples?

**Comment by jon-cochran** [comment]
We use Azure Elastic Jobs for Ola's solutions on Azure SQL.  For managed instance is there a reason not to use SQL Agent?  
If you must use a runbook you should be able to use PowerShell to connect to the managed instance and use `Invoke-Sqlcmd` to execute the maintenance stored procedures.
